packages/vike-react-styled-jsx/Wrapper.tsx

```diff
@@ -7,15 +7,9 @@ import { usePageContext } from 'vike-react/usePageContext'
 function Wrapper({ children }: { children: ReactNode }) {
   const pageContext = usePageContext()
 
-  if (isBrowser() || pageContext.config.styledJsx === null) return <>{children}</>
+  if (pageContext.config.styledJsx === null) return <>{children}</>
 
-  pageContext.config.styledJsx ??= createStyleRegistry()
```

Comment:
Idea: I feel like it would be nice to introduce a new convention.
 -  `pageContext.config.styledJsx` => set by the user 
 - `pageContext.styledJsx.*` => set by vike-react-styled-jsx


In other words: `pageContext.stlyedJsx.registry` instead of `pageContext.stlyedJsxRegistry`.